themes/agnoster.zsh-theme

```diff
@@ -85,6 +85,9 @@ prompt_context() {
 
 # Git: branch/detached head, dirty status
 prompt_git() {
+  if [[ "$(git config --get oh-my-zsh.hide-status 2>/dev/null)" == "1" ]]; then
```

Comment:
I don't think that accomplishes the same thing. That's just testing whether the configuration setting is present, and treats its set values as equivalent.

```
[~/.oh-my-zsh on ⇄ master]
$ if ! git config --get oh-my-zsh.hide-status; then echo yep; fi
yep
[~/.oh-my-zsh on ⇄ master]
$ git config oh-my-zsh.hide-status 0
[~/.oh-my-zsh on ⇄ master]
$ if ! git config --get oh-my-zsh.hide-status; then echo yep; fi
0
[~/.oh-my-zsh on ⇄ master]
$ git config oh-my-zsh.hide-status 1
[~/.oh-my-zsh]
$ if ! git config --get oh-my-zsh.hide-status; then echo yep; fi
1
[~/.oh-my-zsh]
$
```

If you use that test, then it will take the early return for any repo where that config setting is not defined, and it will ignore its value and proceed for any repo where it is defined.